apps/cms/config/config.exs

```diff
@@ -37,4 +37,12 @@ config :cms,
 
 config :cms, :cms_api, CMS.API.HTTPClient
 
-import_config "#{Mix.env()}.exs"
+config :cms, CMS.Repo,
+  conn_opts: [
+    host: "127.0.0.1",
+    port: 6379
+  ],
+  stats: true,
+  telemetry: true
```

Comment:
this will be different for prod. i just don't know how until we can get something running on aws.